server/middleware/rest.js

```diff
@@ -55,14 +56,8 @@ function rest() {
       }
 
       if (app.isAuthEnabled) {
-        // NOTE(bajtos) It would be better to search app.models for a model
-        // of type AccessToken instead of searching all loopback models.
-        // Unfortunately that's not supported now.
-        // Related discussions:
-        // https://github.com/strongloop/loopback/pull/167
-        // https://github.com/strongloop/loopback/commit/f07446a
-        var AccessToken = loopback.getModelByType(loopback.AccessToken);
-        handlers.push(loopback.token({ model: AccessToken }));
+        var AccessToken = registry.getModelByType(loopback.AccessToken);
```

Comment:
You can't use `loopback.AccessToken` here, as it refers to a model registered in the global registry, while `registry.getModelByType` is searching the local per-app registry.

Also are you sure the comment is no longer relevant?